build.gradle

```diff
@@ -18,8 +18,8 @@ repositories {
 }
 
 dependencies {
-  compile "io.netty:netty-handler:4.1.0.Beta8"
-  compile "org.sharegov:mjson:1.3"
+  compile "io.netty:netty-handler:4.1.8.Final"
+  compile "com.google.code.gson:gson:2.8.0"
```

Comment:
good point, sorry, forgot to change that one too. I can make a new pull request but I'm not sure whether that is easier for you.